readme.adoc

```diff
@@ -1,8 +1,8 @@
 :readme:
 :branch: 4.4
 :docs: https://neo4j.com/labs/apoc/4.4
-:apoc-release: 4.4.0.8
-:neo4j-version: 4.4.4
+:apoc-release: 4.4.0.22
```

Comment:
Hmm this is tricky, for 4.4 I guess it does work with just using the previous released version though as there is only one 4.4 branch, unlike 5.x series 🤔